### ISSUE TITLE
I only see the language correction options in English. I need correction spanish

### ISSUE THREAD

**Issue by xtarhx** [giuspen/cherrytree#1946]
**Version, Operative system**
Raspbian

**Describe the bug**
I only see the language correction options in English.

**To Reproduce**
I have looked for a way to try to download the correction language. My default language is Spanish. But for the correction I do not see in preferences or anywhere the option to choose a corrector in Spanish.

**Comment by ciranus** [comment]
> for the correction I do not see in preferences or anywhere the option to choose a corrector in Spanish.

![2022-01-28_21-02](https://user-images.githubusercontent.com/61602511/151613885-69ec9b6c-885e-4a4b-a45d-14093d7d22db.jpg)

+ you need of course to install the dictionary you need (hunspell-es [= spanish] for linux).
Don't know for Raspbian.

**Comment by xtarhx** [comment]
oh thank you very much!!! I just needed to install the dictionary on my raspbian. Install from the terminal and when restarting I was able to use the dictionary.

sudo apt install hunspell-es

**Comment by xtarhx** [comment]
A recommendation, perhaps, is to specify that in the manual. I went crazy looking for it. Perhaps it would be interesting to put the list to install the languages.

**Comment by ciranus** [comment]
> Spell Check Language - Defines which language to use in spell check. The following languages are available

Agreed. Can be improved with "_assuming that dictionaries are installed_", and be more explicit on what is needed to install.

[Cherrytree User Manual - Spell Check Language](https://giuspen.com/cherrytreemanual/#rich-text-pref)
